resources/usa_cultivation/cl_cultivation.lua

```diff
@@ -22,6 +22,23 @@ local KEYS = {
 
 local POLL_INTERVAL_SECONDS = 5
 
+local Whitelisted_Plant_Locations = {
+	{
+		coords = vector3(-8.560081, -2519.72, -9.939499),
+		distance = 20
+	},
+}
+
+local function isAtWhitelistedLocation()
+	for i = 1, #Whitelisted_Plant_Locations do
+        local x, y, z = table.unpack(Whitelisted_Plant_Locations[i].coords)
+        if #(GetEntityCoords(PlayerPedId()) - Whitelisted_Plant_Locations[i].coords) < Whitelisted_Plant_Locations[i].distance then
```

Comment:
```suggestion
        if #(playerCoords - Whitelisted_Plant_Locations[i].coords) < Whitelisted_Plant_Locations[i].distance then
```

since calling natives is CPU intensive we should try to minimize the amount we have to use them

here that means instead of calling `GetEntityCoords` every time in this `for` loop, let's just call it once before the loop and store it in a variable that we can use to check each item against so we only call it one time